app/views/users/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>This is the user ID</h1>
+<section>
+  <div class='card_container'>
+      <img alt='This is an image' src='<%=@user.photo%>' class='user_image'/>
+      <div class='user-info'>
+        <p><%= link_to @user.name, @user, class: 'user-name' %></p>
+        <p>Number of posts: <%= @user.posts_counter%></p>
+      </div>
+    </div>
+</section>
+<article class='bio'>
+  <p>Bio:</p>
+  <p><%=@user.bio%></p>
+</article>
+
+<section>
+<% @posts.each do |post| %>
+  <article class='post-card'>
+    <div>
+      <p><%= post.title%></p>
+      <p><%= post.text%></p>
+    </div>
+  <div class='counters'>
```

Comment:
* Kindly use `#recent_post` method to render the user's 3 most recent posts of a user, you have this methods created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md). to render the user's 3 most recent posts. [as mentioned in the project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.)